### PR TITLE
Added Alias support for Git Bash

### PR DESCRIPTION
**Pull request by abcarcol**

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -135,6 +136,18 @@ function CreatePowershellProfile
     . $PROFILE
 }
 
+function UpdateGitBashProfile
+{
+    $search = (Get-Content $script:bashProfilePath | Select-String -Pattern '# generated by Rancher-Desktop for Alias support in Git Bash').Matches.Success
```
Comment (albertdb) [inline install.ps1:141]:
```suggestion
    $search = (Get-Content $script:bashProfilePath | Select-String -Pattern '# generated by rd-installer for Alias support in Git Bash').Matches.Success
```

File: install.ps1 [hunk 2]
```diff
@@ -135,6 +136,18 @@ function CreatePowershellProfile
...
+    if( -Not $search){
+        Add-Content $script:bashProfilePath "# generated by Rancher-Desktop for Alias support in Git Bash"
```
Comment (albertdb) [inline install.ps1:143]:
```suggestion
        Add-Content $script:bashProfilePath "# generated by rd-installer for Alias support in Git Bash"
```

File: install.ps1 [hunk 3]
```diff
@@ -187,6 +200,19 @@ function ActivateWslVpnkit
 
     Remove-Item wsl-vpnkit.tar.gz -Force
 
+    $search = (Get-Content $PROFILE | Select-String -Pattern '# generated by Rancher-Desktop for VPN support in Powershell').Matches.Success
```
Comment (albertdb) [inline install.ps1:203]:
```suggestion
    $search = (Get-Content $PROFILE | Select-String -Pattern '# generated by rd-installer for VPN support in Powershell').Matches.Success
```

File: install.ps1 [hunk 4]
```diff
@@ -187,6 +200,19 @@ function ActivateWslVpnkit
...
+    if( -Not $search){
+        Add-Content $PROFILE "# generated by Rancher-Desktop for VPN support in Powershell"
```
Comment (albertdb) [inline install.ps1:205]:
```suggestion
        Add-Content $PROFILE "# generated by rd-installer for VPN support in Powershell"
```

File: install.ps1 [hunk 5]
```diff
@@ -187,6 +200,19 @@ function ActivateWslVpnkit
...
+        Add-Content $PROFILE "# Start the VPN support"
+        Add-Content $PROFILE "wsl -d wsl-vpnkit service wsl-vpnkit start 2> `$null"
+    }
+    $search = (Get-Content $script:bashProfilePath | Select-String -Pattern '# generated by Rancher-Desktop for VPN support in Git Bash').Matches.Success
```
Comment (albertdb) [inline install.ps1:209]:
```suggestion
    $search = (Get-Content $script:bashProfilePath | Select-String -Pattern '# generated by rd-installer for VPN support in Git Bash').Matches.Success
```

File: install.ps1 [hunk 6]
```diff
@@ -187,6 +200,19 @@ function ActivateWslVpnkit
...
+    if( -Not $search){
+        Add-Content $script:bashProfilePath "# generated by Rancher-Desktop for VPN support in Git Bash"
```
Comment (albertdb) [inline install.ps1:211]:
```suggestion
        Add-Content $script:bashProfilePath "# generated by rd-installer for VPN support in Git Bash"
```

File: install.ps1 [hunk 7]
```diff
@@ -187,6 +200,19 @@ function ActivateWslVpnkit
...
+        Add-Content $script:bashProfilePath "# Start the VPN support"
+        Add-Content $script:bashProfilePath "wsl -d wsl-vpnkit service wsl-vpnkit start 2> `$null"
```
Comment (albertdb) [inline install.ps1:213]: Shouldn't be `/dev/null`?

**Review by albertdb — CHANGES_REQUESTED** [review]

File: install.ps1 (hunk 7 above)
Comment (abcarcol) [reply install.ps1:213]: Yes, my bad.

**Review by jbellver99 — APPROVED** [review]

File: install.ps1 [hunk 8]
```diff
@@ -187,6 +200,19 @@ function ActivateWslVpnkit
 
     Remove-Item wsl-vpnkit.tar.gz -Force
 
+    $search = (Get-Content $PROFILE | Select-String -Pattern '# generated by rd-installer for VPN support in Powershell').Matches.Success
+    if( -Not $search){
+        Add-Content $PROFILE "# generated by rd-installer for VPN support in Powershell"
+        Add-Content $PROFILE "# Start the VPN support"
+        Add-Content $PROFILE "wsl -d wsl-vpnkit service wsl-vpnkit start 2> `$null"
```
Comment (albertdb) [inline install.ps1:207]: Is the backtick ` expected?

Comment (abcarcol) [reply install.ps1:207]: Yes. it escapes the '$' character

**Pull request merged**